update!: Windows guide (#1659)
privacyguides/privacyguides.org: docs/windows/privacy.md

---
title: Privacy in Windows
icon: material/incognito
---

## Using Microsoft account

> You should never sign-in to Windows with a Microsoft account. Signing-in to applications like Microsoft Office (which some users are required to do for their school or company) will trigger a dark pattern offering you to sign in to Windows, which will connect your device to your Microsoft account, and make it easier to send data to Microsoft servers and it is critical to reject this offer.

Reviewer: ```suggestion
You should never sign in to Windows with a Microsoft account. Signing in to applications like Microsoft Office (which some users are required to do for their school or company) will trigger a dark pattern offering you to sign in to Windows, which will connect your device to your Microsoft account, and make it easier to send data to Microsoft servers and it is critical to reject this offer.
```

Reviewer: I think the "make it easier to send data to Microsoft servers and it is critical to reject this offer." can be rephrased and expanded upon a bit

It’s worth noting that according to [this study](https://www.autoriteitpersoonsgegevens.nl/sites/default/files/atoms/files/public_version_dutch_dpa_informal_translation_summary_of_investigation_report.pdf) it seems that Windows collects more telemetry when signed into a Microsoft Account.

![Using account for specific app](/assets/img/windows/signin-one-app.webp)

You should log in to that specific app only if you need to.

or 

Create another standard user account and connect it to Microsoft account if you are required for School or Work and keep the apps to that account alone. By restricting other data drive access, it is fully isolated from other profiles.

## Telemetry

To disable telemetry at full level, Open Group policy and navigate to `Computer Configuration` > `Administrative Templates` > `Windows Components` > `Data Collection and Preview builds` and choose as required

> ![Disable telemtry](/assets/img/windows/disable-telemetry.webp)

Reviewer: ```suggestion
![Disable telemetry](/assets/img/windows/disable-telemetry.webp)
```

The above works only if you use Enterprise or Education edition. If Professional, It will send required (Basic) data.

If you read this article - [https://www.softscheck.com/en/blog/windows-10-enterprise-telemetry-analysis/](https://www.softscheck.com/en/blog/windows-10-enterprise-telemetry-analysis/), Enterprise even sends data even though telemetry is disabled. But there is no updated info about this available.

> Disabling full telemtry or sending basic data to Microsoft is totally upto the user's threat model.

Reviewer: ```suggestion
Disabling full telemetry or sending basic data to Microsoft is totally up to the user's threat model.
```

> - [ ] Disable `Automatic Sample Submission` in Windows Defender will send your files as a sample for Signature Database and might leak your data. You can do it via the below Group Policy so to not prompt you again and again constantly.

Reviewer: This makes it sound like disabling the option in Defender will send the files.

    ```
    Computer Configuration > Administrative Templates > Windows Components > Microsoft Defender Antivirus > MAPS > Send file samples when further analysis is required to Never Send.
    ```

> - [ ] Disable Windows spotlight by navigating to `User Configuration` > `Administrative Templates` > `Windows Components` > `Cloud Content` and setting **Turn off all Windows Spotlight features** policy to disabled.

Reviewer 2: If you disable this, it will get enabled. If you enable the option, it will get disabled.  Basically the statement here says that you're enabling the spotlight features.   
It is better to read the policy description to not get confused by the options given in the `gpedit.msc`

Author: Yeah, My bad. Thanks for pointing it out. I was experimenting Group Policies back then. So, didn't know what did what.

        !!! note

>         This explicitly disables Windows spotlight features in Lockscreen and Desktop to severe unnecessary between Microsoft servers and the device.

Reviewer: ```suggestion
        This explicitly disables Windows spotlight features in Lockscreen and Desktop to sever unnecessary connections between Microsoft servers and the device.
```

> - [ ] Disable in Bing integration in Windows search, by navigating to `Computer Configuration\Administrative Templates\Windows Components\Search\Don't search the web or display web results`. This way your search queries for local indexed data is not sent to Microsoft.

Reviewer: ```suggestion
- [ ] Disable Bing integration in Windows search, by navigating to `Computer Configuration\Administrative Templates\Windows Components\Search\Don't search the web or display web results`. This way your search queries for local indexed data is not sent to Microsoft.
```
